Python/clinic/sysmodule.c.h

```diff
@@ -1231,7 +1231,7 @@ PyDoc_STRVAR(sys_activate_stack_trampoline__doc__,
 "activate_stack_trampoline($module, backend, /)\n"
 "--\n"
 "\n"
-"Activate the perf profiler trampoline.");
+"Activate stack profiler trampoline *backend*.");
```

Comment:
Should this mention that `"perf"` is the only option too?

Reply:
I'd not bother with that in the docstring.